crates/bevy_render/src/texture/image_texture_loader.rs

```diff
@@ -9,6 +9,32 @@ use bevy_utils::BoxedFuture;
 #[derive(Clone, Default)]
 pub struct ImageTextureLoader;
 
+macro_rules! define_file_extensions {
+    ( $( $s:literal => $v:ident ),+ $(,)? ) => {
+
+        const FILE_EXTENSIONS: &'static [&'static str] = &[
+            $( $s ),+
+        ];
+
+        fn match_file_extension (ext: &str) -> Option<image::ImageFormat>
+        {
+            if false { None } $(
+
+                else if ext.eq_ignore_ascii_case ($s) {
+                    Some(image::ImageFormat::$v)
+                }
+
+            )+ else { None }
+        }
+
+    };
+}
+
+define_file_extensions! {
+    "png" => Png, "dds" => Dds,
+    "tga" => Tga, "jpeg" => Jpeg,
```

Comment:
What about "jpg" without the e?

Reply:
That would be the more common one! Thanks.